Quadriga/src/main/webapp/resources/txt-layout/js/d3networkvisualize.js

```diff
@@ -10,6 +10,10 @@
 
 
 function d3init(graph, networkId, path,type) {
+	
+    var token = '${_csrf.token}';
+    var header = '${_csrf.headerName}';
```

Comment:
these will not be replace. They are static files that are not run through the jsp engine.